src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -50,11 +50,19 @@ import { getSortedGenericAssayTabSpecs } from 'shared/lib/GenericAssayUtils/Gene
 import { HelpWidget } from 'shared/components/HelpWidget/HelpWidget';
 import GroupComparisonPathwayMapper from './pathwayMapper/GroupComparisonPathwayMapper';
 import GroupComparisonMutationsTab from './GroupComparisonMutationsTab';
+import { IPathwayMapperTable } from 'pathway-mapper';
+import PathwayMapperTable from 'shared/lib/pathwayMapper/PathwayMapperTable';
 
 export interface IGroupComparisonPageProps {
     routing: any;
     appStore: AppStore;
 }
+export interface PathwayMapParams {
+    dropDownTitle: string;
+    isPercentageMatch: number;
+    isAlterationEnabled: number;
+    considerOnlyTCGAPanPathways: boolean;
+}
```

Comment:
We can move this into a separate `GroupComparisonPathwayMapperUserSelectionStore` class. Also maybe rename the interface as `IRankingChoices`?